components/schemes/Container/src/MDCContainerScheme.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "MaterialColorScheme.h"
+#import "MaterialShapeScheme.h"
+#import "MaterialTypographyScheme.h"
+
+/**
+ A class conforming to the MDCContainerScheming provides scheme properties for all the
+ theming systems (color, typography, shape, etc.) MDC supports.
+ */
+@protocol MDCContainerScheming
+
+@property(nonatomic, nullable, readonly) id<MDCColorScheming> colorScheme;
```

Comment:
Please add documentation to these properties.

Reply:
I think both the name and the comment listed behind it stating its default value should be enough?

Reply:
That comment isn't visible in generated documentation. Please add documentation comments to the properties.   If the property is documented in the protocol, then we don't usually document it in the class except if it differs somehow.

Reply:
Addressed in da30204eb15d8e8b46631210c33fcb1e74cc13ee